js/.vscode/launch.json

```diff
@@ -6,21 +6,12 @@
   "configurations": [
     {
       "name": "[common] Launch Unit Tests",
-      "args": [
-        "-u",
-        "bdd",
-        "--timeout",
-        "999999",
-        "--colors",
-        "${workspaceFolder}/common/test/**/*.js"
-      ],
+      "args": ["-u", "bdd", "--timeout", "999999", "--colors", "${workspaceFolder}/common/test/**/*.js"],
```

Comment:
>  "args": ["-u", "bdd [](http://example.com/codeflow?start=4&length=21)

This seems to have a formatting problem that we addressed earlier?

Reply:
Recently we have multiple web PRs pending, and some of them passed the CI very long ago. When these PR get merged the CI failed to cache format problems for those introduced after the change of updating the formatter. more details included in https://github.com/microsoft/onnxruntime/pull/16960.

There was an old discussion about how to technically get rid of this situation. the conclusion was that if any new commits merged into main branch will invalidate all existing CI, we will no longer have this kind of issues. however considering the time of the CI jobs, this is almost impossible. so the existing status is the best trade-off so far.

Reply:
I would like this to be all consistent to any style that makes CI work. Currently, I am not seeing it consistent, so some part of is bound to break. When things are inconsistent, then it is not immediately clear if any new changes are correctly formatted or it is the old code that is bad.

Reply:
The formatter is configured to be consistent. Specifically, [Prettier](https://prettier.io/), the JSON/JSONC formatter, working with the determined [config file](https://github.com/microsoft/onnxruntime/blob/main/js/.prettierrc), has a consistent behavior.

The reason why launch.json is not formatted is because of the reason that I described in the PR description: an old PR which passed all CI checks without the latest formatter update. The PR should have merge/rebase to latest main to allow the CI to catch the format issue.